actions/rootio/v1/pkg/storage/partition.go

```diff
@@ -90,7 +90,7 @@ func Partition(d types.Disk) error {
 				End:   sectorEnd,
 			}
 
-			sectorStart += sectorEnd
+			sectorStart = sectorEnd + 1
```

Comment:
This change asserts sectorStart is 1, which is not what was given above. Why change this?

Reply:
This is fix for https://github.com/tinkerbell/hub/issues/98